server/routes/profile.router.js

```diff
@@ -14,12 +14,32 @@ router.get('/', rejectUnauthenticated, (req, res) => {
         });
 });
 
+//returns the bio associated with a particular user, when passed the id of that user
+router.get('/:id', rejectUnauthenticated, (req, res) => {
+    let userID = req.params.id;
+    Profile.findById(userID)
+    .then((result) => {
+        res.send(result.bio);
+    })
+    .catch((error) => {
+        console.log(error);
+        res.sendStatus(500);
+    });
+});
+
 // takes in 'status' param with value 'reset', 'spam', or 'notSpam'
 // takes in body the _id of the profile
 router.put('/:status', rejectUnauthenticated, (req, res) => {
-
+    let statusUpdate = req.params.status;
```

Comment:
Shouldn't this route also update 'flagged' 'isDeleted' and 'reason'?

Reply:
Kind of. I fixed so that isDeleted would be updated appropriately; I didn't see the sense in changing flagged or reason, because it seems useful to maintain a record of the filter's results in either case, right?